client/src/components/applications/profil/DetailPage.js

```diff
@@ -0,0 +1,217 @@
+import React, {Component} from 'react'
+import ReactDOM from 'react-dom'
+import './DetailPage.css'
+
+import * as serverAPI from './actions/serverAPI'
+import Cardboard from './components/Cardboard'
+import DetailKatasterTable from './components/DetailCadastralTable'
+import DetailAssetDeclaration from './components/DetailAssets'
+import MapContainer from './components/MapContainer'
+import {NavLink} from 'react-router-dom'
+import {Row, Col} from 'reactstrap'
+
+class DetailPage extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      politician: {},
```

Comment:
We should use redux where possible so we have only single source of state. If you don't want to pollute redux state use recompose (https://github.com/acdlite/recompose) or data providers for fetches.

Reply:
This will be changed in future PR.